### PR TITLE
Re-adds Valentine's Day Chocolate

### PR DESCRIPTION
**Pull request by Shadowmech88**
Except it only appears in uplinks and merch computers on Valentine's Day.
If one is somehow spawned otherwise, such as via cooking, and it's not Valentine's Day, it gets replaced with a burned mess.

### CODE REVIEW

File: code/game/machinery/computer/store.dm [hunk 1]
```diff
@@ -39,6 +39,12 @@
 
 /obj/machinery/computer/merch/New()
 	..()
+	var/month = time2text(world.realtime,"MM")
+	var/day = time2text(world.realtime,"DD")
+	if(month == "02" && day == "14")
```
Comment (ghost) [inline code/game/machinery/computer/store.dm]: Might be an idea to have a define or macro or something, so we don't have all these month and day variables floating around, and encourage others to have some things locked behind certain dates.

Comment (PJB3005) [reply code/game/machinery/computer/store.dm]: Do `if (time2text(world.realtime, "DD:MM") == "14:02")` instead.

**Comment by Blithering** [comment]
It was a shame to have all the holiday stuff reverted rather than commented out. Automating the readding of it is even better.

File: code/game/machinery/computer/store.dm (hunk 1 above)
Comment (Shadowmech88) [reply code/game/machinery/computer/store.dm]: I'll format it as `"MM/DD"` instead.

Comment (ghost) [reply code/game/machinery/computer/store.dm]: madman

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 2]
```diff
@@ -513,9 +513,7 @@
 
 /obj/item/weapon/reagent_containers/food/snacks/chocolatebar/wrapped/valentine/New()
 	..()
-	var/month = time2text(world.realtime,"MM")
-	var/day = time2text(world.realtime,"DD")
-	if(month != "02" || day != "14")
+	if(time2text(world.realtime, "MM/DD") == "02/14")
```
Comment (ComicIronic) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: `!=`

**Review by ComicIronic — CHANGES_REQUESTED** [review]
Woops

**Review by ComicIronic — APPROVED** [review]

**Comment by SonixApache** [comment]
neat

**Pull request merged**